nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/createindex/CreateIndexFacetImpl.java

```diff
@@ -88,6 +94,43 @@ public void on(RepositoryDeletedEvent deletedEvent) {
     deleteIndexYaml();
   }
 
+  @Subscribe
+  @Guarded(by = STARTED)
+  @AllowConcurrentEvents
+  public void on(AssetCreateEvent created) {
+    log.debug("Updating index.yaml for hosted repository {}", getRepository().getName());
```

Comment:
You can define a constant for `Updating index.yaml for hosted repository {}` statement